tmail-backend/backend-commons/elasticsearch-v6/src/main/java/org/apache/james/backends/es/UpdatedRepresentation.java

```diff
@@ -25,44 +25,26 @@
 import com.google.common.base.MoreObjects;
 import com.google.common.base.Preconditions;
 
-public class UpdatedRepresentation {
-    private final DocumentId id;
-    private final String updatedDocumentPart;
-
-    public UpdatedRepresentation(DocumentId id, String updatedDocumentPart) {
+public record UpdatedRepresentation(DocumentId id, String updatedDocumentPart) {
+    public UpdatedRepresentation {
         Preconditions.checkNotNull(id);
         Preconditions.checkArgument(!Strings.isNullOrEmpty(updatedDocumentPart), "Updated document must be specified");
-        this.id = id;
-        this.updatedDocumentPart = updatedDocumentPart;
-    }
-
-    public DocumentId getId() {
-        return id;
-    }
-
-    public String getUpdatedDocumentPart() {
-        return updatedDocumentPart;
     }
 
     @Override
     public final boolean equals(Object o) {
         if (o instanceof UpdatedRepresentation other) {
             return Objects.equals(id, other.id)
-                && Objects.equals(updatedDocumentPart, other.updatedDocumentPart);
+                    && Objects.equals(updatedDocumentPart, other.updatedDocumentPart);
         }
         return false;
     }
 
-    @Override
-    public final int hashCode() {
-        return Objects.hash(id, updatedDocumentPart);
-    }
-
     @Override
     public String toString() {
         return MoreObjects.toStringHelper(this)
-            .add("id", id)
-            .add("updatedDocumentPart", updatedDocumentPart)
-            .toString();
+                .add("id", id)
+                .add("updatedDocumentPart", updatedDocumentPart)
+                .toString();
```

Comment:
indent issue?

Reply:
weird, git not change on my local 
![image](https://user-images.githubusercontent.com/81145350/227473203-e1ea30f3-75e2-47f8-a4af-4c42cf78d017.png)

Maybe `tab`,`space`..., if I reformat it by linagora setting, it will change more. 

Can we ignore it?